src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -560,6 +560,9 @@
   "com.minecolonies.coremod.gui.townhall.rename.title": "Rename your Colony",
 
   "com.minecolonies.coremod.gui.townhall.addplayer": "Add Player:",
+  "com.minecolonies.coremod.gui.townhall.addrank": "Add Rank:",
+  "com.minecolonies.coremod.gui.townhall.removerank": "Remove Rank",
```

Comment:
Colon here too (to match the above message)?

Reply:
Nah, this is on a button